tests/modules/nf-core/canu/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CANU } from '../../../../modules/nf-core/canu/main.nf'
+
+workflow test_hicanu {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
```

Comment:
`single_end` isnt used at all anyway, as far as I have seen, but wouldn't
```suggestion
        [ id:'test', single_end:true ], // meta map
```
make more sense?

Reply:
Yes, definitely single end! I'll try to remember in future module creation.